libs/sdk/src/profiles/index.ts

```diff
@@ -157,18 +157,24 @@ class AWF_Profile {
       // account: pushAccount,
     });
 
+    const notificationsEnabled = await this.enableBrowserNotifications();
```

Comment:
There are cases in which 'enableBrowserNotifications' function throws an error and those cases are not handled here

Reply:
So there are two potential points of failure:

1. Browser does not support push notifications - we need to trigger notifications through service worker (triggering them via `new Notification()` is no supported in all browsers). I will implement through existing service worker then. 

2. Client not initialised - this error is expected in case our implementation failed to create client instance. Should we use any logger here to note the bug or just let the thrown error bubble up?

Reply:
Let's have a call tomorrow to understand better the issues you have encountered @kunstefix